runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java

```diff
@@ -101,6 +98,15 @@ public Dimension render(Graphics2D graphics)
 
 	private BufferedImage getImage(Item item)
 	{
-		return itemManager.getImage(item.getId(), item.getQuantity(), item.getQuantity() > 1);
+		ItemComposition itemComposition = itemManager.getItemComposition(item.getId());
+		
+		if (itemComposition.isStackable())
```

Comment:
This whole if/else block is `return itemManager.getImage(item.getId(), item.getQuantity(), itemComposition.isStackable());` ?

Reply:
Didn't realize I could just do that. Thanks for letting me know!